### PR TITLE
WebRTC Protobuf & Shared Code Update 

### PR DESCRIPTION
**Pull request by Gaukas**
Adding WebRTC-related fields to protobuf. Changes are not final.

### CODE REVIEW

**Review by Gaukas — COMMENTED** [review]
- Added webrtc capability to shared code.
- Modified Protobuf to support webrtc.

File: protobuf/signalling.proto
Comment (ewust) [inline protobuf/signalling.proto:157]: We should probably not send the shared secret in the clear like that, no? Is this derived from the seed? If so, then no need to send it in this protobuf

File: tapdance/conjure.go
Comment (ewust) [inline tapdance/conjure.go:295]: Probably should not merge this whole PR until we have the rest completed; we can merge the protobuf things separately for now

**Review by ewust — CHANGES_REQUESTED** [review]
Let's trim this down to just the protobuf changes for now, and leave the TODOs/active dev stuff on a branch.

**Pull request closed without merging**
